src/Artsy/Router/buildClientApp.tsx

```diff
@@ -65,6 +66,7 @@ export function buildClientApp(config: RouterConfig): Promise<Resolve> {
           parse: queryStringParsing,
           stringify: qs.stringify,
         }),
+        trackingMiddleware(),
```

Comment:
In the future, other middleware can go here. It conforms to Redux's [specification](https://redux.js.org/advanced/middleware/).